examples/material-next-app-router-ts/src/app/page.tsx

```diff
@@ -1,6 +1,78 @@
 import * as React from 'react';
-import Home from '@/layouts/Home/Home';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Unstable_Grid2';
+import Drawer from '@mui/material/Drawer';
+import Typography from '@mui/material/Typography';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+import MediaCard from '@/components/MediaCard';
 
-export default function RootPage() {
-  return <Home />;
+export default function HomePage() {
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+      }}
+    >
+      <Box>
+        <Alert severity="info" sx={{ mt: 2, mb: 5 }}>
+          <AlertTitle>Hello 👋</AlertTitle>
+          This app uses the Next.js App Router and Material UI v5.
+        </Alert>
+
+        <Grid container rowSpacing={3} columnSpacing={3}>
+          <Grid xs={6}>
+            <MediaCard
+              heading="CMYK"
+              text="The CMYK color model (also known as process color, or four color) is a subtractive color model, based on the CMY color model, used in color printing, and is also used to describe the printing process itself."
+            />
+          </Grid>
+          <Grid xs={6}>
+            <MediaCard
+              heading="HSL and HSV"
+              text="HSL (for hue, saturation, lightness) and HSV (for hue, saturation, value; also known as HSB, for hue, saturation, brightness) are alternative representations of the RGB color model, designed in the 1970s by computer graphics researchers."
```

Comment:
I imagine it was helpful to test the whole PR end-to-end but examples are not templates. Could we revert these changes? Thanks

Reply:
It would be nice to circle back to a conversation we had a couple months ago about standardizing the layout for boilerplate examples. cc @danilo-leal

Reply:
Next.js approach makes a lot of sense for me, once the setup works, they link you a template https://vercel.com/templates?framework=next.js.

@samuelsycamore I believe we already have a standard layout, do you mean improving it and enforcing it to all the examples?

Reply:
Yep that's the idea, standardize something a little more neat and polished like the pretty startup pages used by Vercel, Vite, etc.

Reply:
Adding to the list ⎯ I think this adds tons of value! 👍 Also, having a global "Templates" page, complementary to the individual ones found in each product's docs, also looks like a good idea!

Reply:
I added the idea to https://www.notion.so/mui-org/example-Simplify-them-to-link-to-templates-546e5a4574ae4a32b0e684ad1c1a54d0